hack/vagrant/Vagrantfile

```diff
@@ -0,0 +1,22 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+
+# refer: https://github.com/hashicorp/vagrant/issues/9442
+Vagrant::DEFAULT_SERVER_URL.replace('https://vagrantcloud.com')
+
+Vagrant.configure("2") do |config|
+  config.vm.boot_timeout=600
+  # ubuntu/xenial64 is from official ubuntu 16.04 LTS (Xenial Xerus) daily build, which is 
+  # based on the long-term supported Linux release series 4.4, refer: https://app.vagrantup.com/ubuntu
+  config.vm.box = "ubuntu/xenial64"
+  config.vm.provision :shell, path: "bootstrap.sh"
+  config.vm.synced_folder "../../", "/go/src/github.com/alibaba/pouch"
+  
+  config.vm.define "pouch-dev-node" do |node|
+    node.vm.network "private_network", ip: "172.30.30.10"
```

Comment:
Hi, why we need this ip, here? we can use `vagrant ssh` to login, right?

Reply:
Basically, the static IP address is not good. It could cause conflict. If all the envs are the same, that will be fine. However, every dev-env is unique and under different sub-net. 

Therefore, I think we can remove this configure. VirtualBox supports port forwarding so that we can use this to do things which you mentions.

How do you think?

Reply:
You are right, i'll remove this configure and keep it simple. It can be added if then we need a cluster setting like https://github.com/docker/libnetwork/blob/master/Vagrantfile or https://github.com/pires/kubernetes-vagrant-coreos-cluster/blob/master/Vagrantfile.

Reply:
Sure thing!!